plugins/catalog/src/components/EntityPageLayout/EntityPageLayout.tsx

```diff
@@ -159,6 +164,14 @@ export const EntityPageLayout = ({ children }: PropsWithChildren<{}>) => {
         </Content>
       )}
 
+      {entity && (
+        <EntityBadgesDialog
```

Comment:
Let's create an issue around how we can refactor this so that it's added from the app rather than imported by the catalog plugin